parm/config/gfs/config.resources

```diff
@@ -885,7 +886,7 @@ elif [[ ${step} = "epos" ]]; then
     export npe_epos=80
     export nth_epos=4
     if [[ "${machine}" == "HERA" ]]; then
-      export nth_epos=6
+      export nth_epos=1
```

Comment:
Should this be changed to 1 for all machines? We can get rid of this special case for Hera and just make it 1 everywhere.

Reply:
Probably but I did not test and am not sure why this was 4 and 6 in the first place... It seems like it is not using any threads at all.

Reply:
Looks like Kate _lowered_ it to 4 after the WCOSS2 port (PR #1070). And before that it had been lowered to 6 from 8 pre-GitHub. Who knows why it was so high to begin with (I would assume memory). Given Hera is the slowest, lowest-mem system, I suspect lowering it everywhere would work.

Reply:
With Russ's reply, I'm more confident we can just use 1.

Reply:
Done.